prodeus/prodeus.asl

```diff
@@ -30,61 +30,50 @@ startup
 
 init
 {
-    vars.Unity.TryOnLoad = (Func<dynamic, bool>)(helper =>
+    vars.Helper.TryLoad = (Func<dynamic, bool>)(mono =>
     {
         // Get references to classes
-        var loadingInfo = helper.GetClass("Assembly-CSharp", "LoadingInfo");
-        var gameInfo = helper.GetClass("Assembly-CSharp", "GameInfo");
-        var mapInfo = helper.GetClass("Assembly-CSharp", "MapInfo");
+        var loadingInfo = mono["LoadingInfo"];
+        var gameInfo = mono["GameInfo"];
+        var mapInfo = mono["MapInfo"];
 
         // Loading data
-        vars.Unity.Make<IntPtr>(loadingInfo.Static, loadingInfo["instance"]).Name = "loadingInfoInstance";
+        vars.Helper["loadingInfoInstance"] = loadingInfo.Make<IntPtr>("instance");
```

Comment:
Am I missing something here? Why not just `vars.Helper["loading"] = loadingInfo.Make<bool>("instance", "loading");`?

Reply:
Not sure. Maybe the class isn't loaded in initially - I just left it how it was. A question for @samontenegro.

Reply:
Hey! This was with the old lib, where I had problems accessing the class field directly since Prodeus is an IL2CPP game. The `loading` field is not static, and I had to offset into it. 

I can take a look at this.

Reply:
Yup, this looks solid. @mitchell-merry did you test the first few levels? 🙂 If you saw everything good in-game lmk and I'll approve the PR.

Reply:
I think I did, but let me triple check for you now.

Reply:
It works, & I can make the change Ero suggested for you as well if you want. I'm not sure what you mean - you could always access non-static members of an instance, even in old versions of the library, using `loadingInfo["loading"]` I believe.